package.json

```diff
@@ -23,7 +22,6 @@
     "react-intl": "^6.3.0",
     "react-native": "0.71.3",
     "react-native-gesture-handler": "~2.9.0",
-    "react-native-reanimated": "~2.14.4",
```

Comment:
to my knowledge, this was a peer dep for `@react-navigation/drawer`, so cautiously removing